AK/Variant.h

```diff
@@ -418,6 +418,15 @@ struct Variant
         return index_of<T>() == m_index;
```

Comment:
Typo in the commit message: Varient -> Variant

Reply:
done!